idaes/models_extra/gas_solid_contactors/unit_models/tests/test_FB0D.py

```diff
@@ -345,31 +365,49 @@ def test_scaling(self, iron_oc):
             ) == iscale.get_constraint_transform_applied_scaling_factor(c)
         for t, c in FB0D.solids_energy_holdup_constraints.items():
             assert pytest.approx(
-                1.27323E-9, abs=1e-11
+                1.27324E-7, abs=1e-9
```

Comment:
This is a very loose tolerance to be checking too - generally you should check to near solver tolerance, thus I tend to use `rel=1e-5`.